tools/hmac/HMACValidatorPayments.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+// script to calculate the HMAC signature of Payments webhooks (where the signature is calculated considering
+// a subset of the fields in the payload - i.e. NotificationRequestItem object)
+//
+// Run with: `php tools/hmac/HMACValidatorPayments.php {hmacKey} {path to JSON file}
+// php tools/hmac/HMACValidatorPayments.php 11223344D785FBAE710E7F943F307971BB61B21281C98C9129B3D4018A57B2EB tools/hmac/payload.json
+
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+use Adyen\Util\HmacSignature;
+
+if ($argc !== 3) {
```

Comment:
Check if the arguments are correct, 4?

Reply:
Verified, that argument check is correct